src/main/java/com/google/sps/models/TopChartsCluster.java

```diff
@@ -1,18 +1,24 @@
 package com.google.sps.servlets;
 import java.util.ArrayList;
 
-public class TopChartsCluster implements Cluster {
+public class TopChartsCluster extends Cluster {
     private String title = "Top Charts";
+    private String subtitle = "For Wear OS";
     private ArrayList<Chart> charts = new ArrayList<Chart>();
```

Comment:
Can you also update this to use List 

`private List<Chart> charts = new ArrayList<Chart>();`